src/git/README.md

```diff
@@ -22,7 +22,7 @@ Install an up-to-date version of Git, built from source as needed. Useful for wh
 
 ## OS Support
 
-This Feature should work on recent versions of Debian/Ubuntu-based distributions with the `apt` package manager installed.
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

Reply:
Will do.

Reply:
Commit `cba4c71` resolves this request.